admin/test/test_packaging.py

```diff
@@ -542,19 +542,16 @@ def test_version_found(self):
         """
         ``GetPackageVersion`` assigns the exact version of a found package to
         its ``version`` attribute.
-
-        In particular, newer versions of pip/setuptools normalize the version
-        accoding to PEP440. We aren't prepared to handle that yet.
         """
         versions = [
             '0.3.2',
             '0.3.3dev5',
-            '0.3.2+doc1',
-            '0.3.2-1-gf661a6a',
-            '0.3.2+doc1-1-gf661a6a',
+            '0.3.2+post1',
```

Comment:
`0.3.2+post1` and alike are not valid versions - replace `+post` with `.post` wherever it exists.